device/accton/x86_64-accton_as7716_32x-r0/Accton-AS7716-32X/sai.profile

```diff
@@ -0,0 +1,2 @@
+SAI_INIT_CONFIG_FILE=/etc/bcm/th-as7716-32x100G.config.bcm
```

Comment:
@stcheng, the deletion of th-as7716-32x100G.config.bcm was done and we sent it to via your hotmail e-mail account.